packages/select/src/vaadin-select-value-button.js

```diff
@@ -23,7 +24,7 @@ registerStyles(
 
     /* placeholder styles */
     ::slotted(:not([slot]):not([selected])) {
-      line-height: normal;
+      line-height: 1;
```

Comment:
For some reason, using `normal` on Windows causes the input container height to increase by 1 pixel.
This change caused the placeholder screenshot to be updated for Lumo.

Reply:
Are font descenders still visible with this change, for example, `j` and `g`?

Reply:
Can confirm that this works, single-line and multiline.